configtx/msp.go

```diff
@@ -139,6 +145,20 @@ func (m *MSP) isCACert(signingCert *x509.Certificate) error {
 	return fmt.Errorf("signing cert is not a root/intermediate cert for this MSP: %s", m.Name)
 }
 
+func (m *MSP) setMSPConfig(configGroup *cb.ConfigGroup) error {
```

Comment:
nit: This function naming is a bit odd. You would expect a `setMSPConfig` function to take an `mspConfig` in as a parameter. I'm fine leaving it as is but we may want to consider renaming this since it can get twisted/confusing.